internal/zinc-apiinfo/src/main/scala/xsbt/api/APIUtil.scala

```diff
@@ -13,7 +13,10 @@ object APIUtil {
   val modifiersToByte = (m: Modifiers) => {
     import m._
     def x(b: Boolean, bit: Int) = if (b) 1 << bit else 0
-    (x(isAbstract, 0) | x(isOverride, 1) | x(isFinal, 2) | x(isSealed, 3) | x(isImplicit, 4) | x(isLazy, 5) | x(isMacro, 6)).toByte
+    (x(isAbstract, 0) | x(isOverride, 1) | x(isFinal, 2) | x(isSealed, 3) | x(isImplicit, 4) | x(
+      isLazy,
+      5
+    ) | x(isMacro, 6)).toByte
```

Comment:
Oh I know the problem. I'm sure over the coming time while I work on a codebase with scalafmt enforcement (:tada:) I'll get motivated enough to try and contribute an improvements in this area to scalafmt :)